collector/configuration/liverig.properties.md

```diff
@@ -68,7 +68,14 @@
 | `enable_witsml_tree_event` <br> **This Property was removed on Liverig 2.22.0**                                                                                                                                                                                                                                                                    | `boolean`  | `true`                               | 2.17.0 |           |                                                                                                          |                          |
 | `witsml_tree_event_period_seconds` <br> **This Property was removed on Liverig 2.22.0**                                                                                                                                                                                                                                                            |   `long`   | `600`                                | 2.17.0 |     s     |                                                                                                          |                          |
 | `wildcard_period_in_seconds` <br> Sets the Wildcard Period if the Request does not contain it.                                                                                                                                                                                                                                                     |   `int`    | `600`                                | 2.22.0 |     s     |                                                                                                          |                          |
-| `enable_revision`                                                                                                                                                                                                                                                                                                                                  | `boolean`  | `false`                              | 2.24.1 |           |                                                                                                          |                          |
+| `witsml_change_mechanism_feature` <br> Enable/Disable WITSML change mechanism. | `boolean` | `true` | 5.17.x |||||
```

Comment:
The default value is not true as far I know. Please fix this.

Reply:
Changed to _false_